### PR TITLE
Runtime ESC count / gpio / pin detection

### PR DESCRIPTION
**Pull request by 4712**
Changed disable beeper method.
Fix indentation level to 4.
Removed CLI support for 1wire pass through

### CODE REVIEW

**Comment by hydra** [comment]
please rebase this from current master branch and I'll merge it.

File: src/main/drivers/pwm_mapping.h [hunk 1]
```diff
@@ -77,9 +79,27 @@ typedef struct drv_pwm_config_s {
 } drv_pwm_config_t;
 
 
+typedef enum {
+  PWM_PF_NONE = 0,
```
Comment (hydra) [inline src/main/drivers/pwm_mapping.h]: indentation incorrect, already fixed in master

File: src/main/io/serial_1wire.c [hunk 2]
```diff
@@ -154,104 +130,87 @@ static void gpioSetOne(uint32_t escIndex, GPIO_Mode mode) {
...
 // This method translates 2 wires (a tx and rx line) to 1 wire, by letting the
 // RX line control when data should be read or written from the single line
 void usb1WirePassthrough(uint8_t escIndex)
 {
-#ifdef STM32F3DISCOVERY
-  ledInitDebug();
+#ifdef BEEPER
+    // fix for buzzer often starts beeping continuously when the ESCs are read
+    // switch beeper silent here
+    beeperSilence();
```
Comment (hydra) [inline src/main/io/serial_1wire.c]: aha! I take it this works ok then?

Comment (4712) [inline src/main/io/serial_1wire.c]: ledInitDebug() does not make sense for me. LED0 and LED1 are assigned for the board in target.h

File: src/main/drivers/pwm_mapping.h (hunk 1 above)
Comment (4712) [inline src/main/drivers/pwm_mapping.h]: Fixed now?

**Comment by 4712** [comment]
Sorry. No other idea how to resolve the conflicts of the rebase than to delete my remote rep, fork again and edit the stiff again locally... I spend more time in playing with github command than is healthy for me...

**Pull request closed without merging**